### PR TITLE
Pagination deeplink

### PR DESCRIPTION
**Pull request by ferrarirosso**
Fixes value mismatch when using deeplink in conjunction with filter
See https://github.com/microsoft-search/pnp-modern-search/issues/2278

### CODE REVIEW

**Comment by ferrarirosso** [comment]
I have the bad habit to use the format document functionality of VS Code
I did only add a call to this.getFiltersDeepLink() in the componentDidUpdate method.

**Comment by wobba** [comment]
Can you update the yml file.. as it should not be part of your PR.

**Comment by wobba** [comment]
Thank you :)

File: search-parts/src/webparts/searchFilters/components/SearchFiltersContainer.tsx [hunk 1]
```diff
@@ -134,13 +134,15 @@ export default class SearchFiltersContainer extends React.Component<ISearchFilte
...
-        this.setState({
-          submittedFilters: submittedFilters
-        });
+      const submittedFilters = this.getSelectedFiltersFromUIFilters(this.state.currentUiFilters);
+
+      this.setState({
+        submittedFilters: submittedFilters
+      });
+
+      this.getFiltersDeepLink();
```
Comment (edarroudi) [inline search-parts/src/webparts/searchFilters/components/SearchFiltersContainer.tsx:145]: this will result in count = 0 even if there is a count

Comment (wobba) [reply search-parts/src/webparts/searchFilters/components/SearchFiltersContainer.tsx:145]: @edarroudi can you propose a fix?

Comment (edarroudi) [reply search-parts/src/webparts/searchFilters/components/SearchFiltersContainer.tsx:145]: added a PR in #2548 . Moving it up lets the rest of the code run an update on the count.

**Pull request merged**